openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.interface.ts

```diff
@@ -13,3 +13,9 @@ export interface AddGlossaryProps {
   onSave: (data: CreateGlossary) => void;
   fetchTags?: () => void;
 }
+
+export enum AddGlossaryError {
+  NAME_REQUIRED = 'name required',
+  NAME_INVALID = 'name invalid',
+  DESCRIPTION_REQUIRED = 'description required',
```

Comment:
Localisation here too

Reply:
this does not require localization as this is the enum keys, which we are using to get localized message from switch case!